.circleci/config.yml

```diff
@@ -13,6 +13,28 @@ parameters:
 
 jobs:
   validate:
+    executor:
+      name: hmpps/java
+      tag: "21.0"
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - gradle-{{ checksum "build.gradle.kts" }}
+            - gradle-
+      - run:
+          command: ./gradlew check -x test
```

Comment:
Exclude tests from the validate job so we can run the tests in multiple parallel executors